index.d.ts

```diff
@@ -618,7 +618,7 @@ export interface CheckboxOwnProps {
 export type CheckboxProps = PolymorphicBoxProps<'input', CheckboxOwnProps>
 export declare const Checkbox: BoxComponent<CheckboxOwnProps, 'input'>
 
-export type CodeOwnProps = TextOwnProps
+export type CodeOwnProps = TextOwnProps & { appearance?: 'default' | 'minimal' }
```

Comment:
Worth lifting this up to `type CodeAppearance = DefaultApperance | 'minimal'` at the top of the file?

Reply:
Sure!